marge/bot.py

```diff
@@ -61,8 +64,18 @@ def _run(self, repo_manager):
         while True:
             log.info('Finding out my current projects...')
             my_projects = Project.fetch_all_mine(self._api)
-
-            for project in my_projects:
+            filtered_projects = [p for p in my_projects if self.project_regexp.match(p.path_with_namespace)]
+            filtered_out = set(my_projects) - set(filtered_projects)
+            if filtered_out:
+                log.info(
+                    'Projects that match project_regexp: %s',
+                    [p.path_with_namespace for p in filtered_projects]
+                )
+                log.info(
+                    'Projects that do not match project_regexp: %s',
+                    [p.path_with_namespace for p in filtered_out]
+                )
```

Comment:
Sorry, I don't follow here. This is currently going to log the filtered out projects continuously (it is under a `while True`) and because it is very unlikely that a project is renamed, all except the first occurrence will be redundant. I would just put everything from `filtered_out = ...` under an `if self._debug:` or something like that.